lib/chef/platform/query_helpers.rb

```diff
@@ -17,11 +17,14 @@
 #
 
 require "chef-utils" unless defined?(ChefUtils::CANARY)
+require_relative "../mixin/powershell_exec"
```

Comment:
We probably don't want to load this on non-Windows platforms, which is why this was guarded to where it was.